vimeo-networking/src/main/java/com/vimeo/networking/model/Privacy.java

```diff
@@ -118,6 +119,9 @@ public static EmbedValue privacyValueFromString(@Nullable final String string) {
     @UseStag
     public enum ViewValue {
 
+        @SerializedName(PRIVACY_STOCK)
+        STOCK(PRIVACY_STOCK),
```

Comment:
This enum value should also be added to `com.vimeo.networking2.enums.ViewPrivacyType`.